client/homescreen/activity-panel/index.js

```diff
@@ -69,6 +70,13 @@ export const ActivityPanel = () => {
 		return null;
 	}
 
+	const getInitialState = ( panelId ) => {
```

Comment:
Nit: how about `getInitialOpenState`, since this is just meant for the `initialOpen` prop and not component state.

Reply:
Sounds good to me. I changed it in the commit da925e6